docs/cookbook/persistence/memory.md

```diff
@@ -0,0 +1,27 @@
+# In-Memory Wallet
+
+The simplest wallet is one that does not have any persistence. All information about the wallet is held in memory, and will be destroyed upon termination of the process.
+
+This means that the addresses the wallet has revealed, the syncing that has been performed including data on the transaction graph will not persist when the wallet is destroyed, and related operations will need to be performed again.
+
+In general, this means performing a `full_sync()` when starting the wallet, because it has no knowledge of which addresses were given out and which scripts still have balances.
+
+See our page on the [difference between full_sync() and scan()]() for more on this topic.
```

Comment:
```suggestion
See our page on the [difference between `full_sync()` and `scan()`]() for more on this topic.
```

Reply:
I'm switching the text to speak of full scan and sync in general, rather than the methods themselves. Let me know what you think. I just felt like there were too many places where these were mentioned, and it was better to write them out as concepts rather than straight methods everywhere.

Reply:
Yes, concepts are better.